app/extensions/helper/helpers.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
I have used laravel logic, we do have getUrl in misc and i did not want to mix the two.